feature-libs/product/configuration/src/textfield-configurator/state/reducers/configurator-textfield.reducer.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { ConfiguratorTextfield } from '../../../../model/configurator-textfield.model';
+import { ConfiguratorTextfield } from '../../model/configurator-textfield.model';
```

Comment:
GH doesn't let me comment on the reducer itself:
On line 24 in `feature-libs/product/configuration/src/textfield-configurator/state/reducers/configurator-textfield.reducer.ts`, instead of `content: content`, you just use a shortcut `content`.

In addition, we can inline the state creation, just by doing:
```ts
return { ...action.payload }
```

Reply:
return {
        content: { ...action.payload },
      };